ideascube/monitoring/tests/test_views.py

```diff
@@ -325,6 +358,48 @@ def test_cant_create_inventoryspecimen_by_barcode_twice(staffapp):
                                             specimen=specimen).count() == 1
 
 
+def test_staff_can_create_inventoryspecimen_by_serial(staffapp):
```

Comment:
Ho, yes. Missing this test.

In a more general way, it's missing some tests mixing barcode and serial.
However, I'm not sure of what we must do in this case.
The original need is to be able to make inventory using serial to not have to print a barcode.
So, do we must handle the case of a user mixing the use-cases ? @barbayellow  ?